### PR TITLE
FilterSetUI Range Value formatting + CaseDetailPane height/state adjustment

### PR DESCRIPTION
**Pull request by alexkb0009**
Screenshot:

### CODE REVIEW

**Review by Bianca-Morris — APPROVED** [review]

**Review by Bianca-Morris — CHANGES_REQUESTED** [review]
Submitted that review too quickly. Just tested on more SV items and ran into this error on selecting filtering tab for [/cases/GAPCA2ORBW3Y/#case-info.filtering](/cases/GAPCA2ORBW3Y/#case-info.filtering), the UDN case (0339-UDN452807-PROBAND-PB-S1-PERIPHERAL_BLOOD-1):

```
FilterSetController changed state: selectedFilterBlockIndices {} {} 10
FilteringTableFilterSetUI.js:1094 Uncaught ReferenceError: qs is not defined
    at eval (FilteringTableFilterSetUI.js:1094)
    at Array.forEach (<anonymous>)
    at eval (FilteringTableFilterSetUI.js:1045)
    at mountMemo (react-dom.development.js:15442)
    at Object.useMemo (react-dom.development.js:15738)
    at useMemo (react.development.js:1521)
    at FieldBlocks (FilteringTableFilterSetUI.js:1040)
    at renderWithHooks (react-dom.development.js:14803)
    at mountIndeterminateComponent (react-dom.development.js:17482)
    at beginWork (react-dom.development.js:18596)

The above error occurred in the <FieldBlocks> component:
    in FieldBlocks
    in div
    in Unknown
    in div
    in Unknown (created by FilteringTableFilterSetUI)
    in div (created by FilteringTableFilterSetUI)
    in Transition (created by ForwardRef)
    in ForwardRef (created by FilteringTableFilterSetUI)
    in div (created by FilteringTableFilterSetUI)
    in div (created by FilteringTableFilterSetUI)
    in FilteringTableFilterSetUI
    in FilterSetController
    in FetchedItem
    in SortController (created by EmbeddedSearchView)
    in CustomColumnController (created by EmbeddedSearchView)
    in ColumnCombiner (created by EmbeddedSearchView)
    in VirtualHrefController (created by EmbeddedSearchView)
    in div (created by EmbeddedSearchView)
    in EmbeddedSearchView (created by EmbeddedItemSearchTable)
    in div (created by EmbeddedItemSearchTable)
    in EmbeddedItemSearchTable (created by CaseViewEmbeddedVariantSampleSearchTable)
    in CaseViewEmbeddedVariantSampleSearchTable
    in Unknown
    in TabPaneErrorBoundary (created by DotRouter)
    in div (created by DotRouter)
    in div (created by DotRouter)
    in DotRouter
    in Unknown (created by ForwardRef)
    in TabPaneErrorBoundary (created by TabPane)
    in div (created by TabPane)
    in TabPane (created by TabView)
    in TabView (created by ForwardRef)
    in ForwardRef (created by CaseView)
    in Unknown (created by CaseView)
    in CurrentFamilyController (created by CaseView)
    in PedigreeVizLoader (created by CaseView)
    in div (created by CaseView)
    in CaseView
    in ContentErrorBoundary
    in Unknown (created by BodyElement)
    in ContentErrorBoundary (created by BodyElement)
    in div (created by BodyElement)
    in div (created by BodyElement)
    in div (created by BodyElement)
    in div (created by BodyElement)
    in body (created by BodyElement)
    in BodyElement (created by App)
    in StrictMode (created by App)
    in html (created by App)
    in App (created by Context.Consumer)
    in Connect(App)
    in Provider

React will try to recreate this component tree from scratch using the error boundary you provided, TabPaneErrorBoundary.
```
    
Might be worth pulling down that insert data and testing locally.

File: src/encoded/static/components/item-pages/CaseView/FilteringTableFilterSetUI.js [hunk 1]
```diff
@@ -853,29 +824,61 @@ function FieldBlocks({ filterBlock, facetDict, schemas }) {
...
+            });
 
             // Merge, e.g. if a from and a to
             if (typeof qs[field] !== "undefined") {
-                qs[field] = qs[field].concat(v);
+                termQs[field] = qs[field].concat(v);
             } else {
-                qs[field] = v;
+                termQs[field] = v;
             }
```
Comment (Bianca-Morris) [inline src/encoded/static/components/item-pages/CaseView/FilteringTableFilterSetUI.js:852]: I think this is the source of the bugginess. Should be `termQs` maybe?

Comment (alexkb0009) [reply src/encoded/static/components/item-pages/CaseView/FilteringTableFilterSetUI.js:852]: Thanks!

**Review by ugur-camoglu — APPROVED** [review]

**Review by Bianca-Morris — APPROVED** [review]
Looks good to me, too.

I noticed one thing with the min and max values for the range facets, though, that we may want to change/check on in cgap-portal repo (there's another semi-related thing, but including that in the SPC review because I think it's more relevant to that code now):
- https://gyazo.com/75e1018d37548326923055dcd8c516f4 Min and max values aren't showing up for a bunch of the range facets... seems to be the ones we've had schema regeneration issues with, so it's possible these got erased from schema or something.

**Pull request merged**